docs/REST API/rest-api.md

```diff
@@ -9205,7 +9307,7 @@ To get the results of an App Store app install, use the [List MDM commands](#lis
 
 #### Example
 
-`GET /api/v1/fleet/software/install/results/b15ce221-e22e-4c6a-afe7-5b3400a017da`
+`GET /api/v1/fleet/software/b15ce221-e22e-4c6a-afe7-5b3400a017da/install/results`
```

Comment:
Proposed breaking change

Reply:
I think in this case the install UUID should actually come _after_ `software/install`, to follow the convention of the ID coming directly after the thing it's identifying.

```suggestion
`GET /api/v1/fleet/software/install/b15ce221-e22e-4c6a-afe7-5b3400a017da/results`
```

Reply:
@rachaelshaw We do same thing for scripts [here](https://fleetdm.com/docs/rest-api/rest-api#get-script-result): `GET /api/v1/fleet/scripts/results/:execution_id`.

Reply:
cc @getvictor

Reply:
TODO: Revert back

Reply:
@rachaelshaw just committed your suggested change (https://github.com/fleetdm/fleet/pull/21548/commits/2c5cfc753d7e29b4e648a849b755cc0d86ebb4a5)